lib/spack/docs/getting_started.rst

```diff
@@ -17,7 +17,8 @@ Spack has the following minimum requirements, which must be installed
 before Spack is run:
 
 #. Python 2 (2.6 or 2.7) or 3 (3.5 - 3.9) to run Spack
-#. A C/C++ compiler for building
+#. A C/C++ compiler for building and the ``bash`` shell for Spack's compiler
+   wrapper
```

Comment:
Add a note here that you only need it for building; it doesn’t have to be your shell. For example, you can run spack in containers where the root shell is dash or ash. You just can’t build at the moment without bash installed.